mlir/lib/Conversion/BufferizationToMemRef/BufferizationToMemRef.cpp

```diff
@@ -132,27 +132,30 @@ struct BufferizationToMemRefPass
       return;
     }
 
-    func::FuncOp helperFuncOp;
+    llvm::DenseMap<Operation *, func::FuncOp> deallocHelperFuncMap;
```

Comment:
nit: Can you add a type alias for this? (With a comment that the `Operation*` is a symbol table.) I think it would improve the readability a bit.

```c++
using DeallocHelperMap = llvm::DenseMap<Operation *, func::FuncOp>;
```

Reply:
DONE